hack/tools/release/notes.go

```diff
@@ -73,6 +73,8 @@ var (
 
 	prefixAreaLabel = flag.Bool("prefix-area-label", true, "If enabled, will prefix the area label.")
 
+	preReleaseVersion           = flag.Bool("pre-release-version", false, "If enabled, will add a pre-release warning header. (default false)")
```

Comment:
Is this being set locally? Would there be a way to understand which tag we're running against and set this flag automatically? For example: v1.0.0-beta.1 I'd expect the same behavior, and similar for alpha.X or rc.X

Reply:
Ah we talked about this here: https://github.com/kubernetes-sigs/cluster-api/pull/9247#discussion_r1300079613 it's doable long-term with a bigger refactor, but we're mostly just trying to make sure we have the standardized formatting somewhere